src/services/countries.js

```
@@ -0,0 +1,16 @@
+const axios = require('axios');
+
+const getCountries = async () => {
+  try {
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)